Constants/ColorConstants.swift

```diff
@@ -37,6 +37,12 @@ struct AppColor {
 
   struct ThemeColors {
     static let DarkWhite = UIColor(red: 250/255, green: 250/255, blue: 250/255, alpha: 1)
+    static let Blue = UIColor(red: 23/255, green: 107/255, blue: 204/255, alpha: 1)
     static let White = UIColor.white
   }
+
+  struct cgColor {
```

Comment:
You can get cgColor from a UIColor, I think this struct is a bad idea because it's a duplicate

Reply:
For some views I need custom cgColor, we can't get them without defining a variable. I can directly create a variable where I need the color. So I wonder need I to create a constant or redefine it when I need it?